site/content/en/docs/contrib/triage.md

```diff
@@ -16,21 +16,23 @@ Triage is an important part of maintaining the health of the minikube repo.
 A well organized repo allows maintainers to prioritize feature requests, fix bugs, and respond to users facing difficulty with the tool as quickly as possible.
 
 Triage includes:
+
 - Labeling issues
 - Responding to issues
 - Closing issues
 
 If you're interested in helping out with minikube triage, this doc covers the basics of doing so.
 
-Additionally, if you'd be interested in participating in our weekly triage meeting, please fill out this [form](https://forms.gle/vNtWZSWXqeYaaNbU9) to express interest. Thank you! 
+Additionally, if you'd be interested in participating in our weekly triage meeting, please fill out this [form](https://forms.gle/vNtWZSWXqeYaaNbU9) to express interest. Thank you!
 
 # Daily Triage
+
 Daily triage has two goals:
 
 1. Responsiveness for new issues
 1. Responsiveness when explicitly requested information was provided
 
-The list of outstanding items are at http://tinyurl.com/mk-tparty/daily-triage - it covers:
+The list of outstanding items are at http://tinyurl.com/mk-tparty/daily-triage- it covers:
```

Comment:
I think the whitespace around this dash should still be there, maybe change to two dashes to that it's clear:

`trage -- it covers`